longest_palindromic_substring/longestPalindromicSubstring.js

```diff
@@ -0,0 +1,46 @@
+function expandAroundCenter (test, left, right) {
+	/*
+	:param test: Input string whose substrings are to be checked
+	:param left: Left end of result palindromic substring
+	:param right: Right end of result palindromic substring
+	:return: Length of palindromic substring
+	*/
+	let n = test.length;
+	while (left >= 0 && right < n && test.charAt(left) === test.charAt(right)) {
+		left--;
+		right++;
+	}
+	return right - left - 1;
+}
+
+function longestPalindromicSubstring (test) {
+	/*
+	Function to find longest substring which is a palindrome
+
+	:param test: Input string whose substrings are to be checked
+	:return: Longest substring of input string which is a palindrome
+
+	Time complexity: O(n^2)
+	Space complexity: O(1)
+	*/
+	let start = 0;
+	let end = 0;
+
+	for (let i = 0; i < test.length; i++) {
+		let length = Math.max(expandAroundCenter(test, i, i)
+													, expandAroundCenter(test, i, i + 1));
```

Comment:
This comma would look good in above line :P same about line 43. Also update your branch.